code/core/src/cli/automigrate.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  AutofixOptionsFromCLI,
+  FixesIDs,
+  allFixes,
+} from '../../../lib/cli-storybook/src/automigrate/fixes';
+
+/**
+ * Get the command to run an automigration
+ *
+ * @example
+ *
+ * ```ts
+ * const command = getAutomigrateCommand('addon-a11y-parameters', {
+ *   yes: true,
+ *   configDir: 'config',
+ *   packageManager: 'npm',
+ * });
+ * // ['storybook', 'automigrate', 'addon-a11y-parameters', '--yes', '--config-dir', 'config', '--package-manager', 'npm']
+ * ```
+ */
+export const getAutomigrateCommand = (
```

Comment:
wouldn't it be better to have an abstraction to run Storybook CLI commands instead? And force the configDir to be passed? The problem we face happens on all Storybook CLI commands essentially, including the Storybook doctor command.

I guess it begs the question too, do we need to spawn npm commands in all places that currently do that? Or can we just call the function if the import is available? I understand it might not work in a few places like when executing something from @storybook/cli as part of create-storybook, but in other places it might work?

Reply:
Let's give it a try!